transform/transformers/ImageTransformer.py

```diff
@@ -1,43 +1,69 @@
-import os
-import glob
-import subprocess
-import zipfile
+#!/usr/bin/env python
+#   coding: UTF-8
+
+import argparse
 import datetime
 import dateutil.parser
-from transform import settings
-import shutil
+import glob
 from io import BytesIO
-from .PDFTransformer import PDFTransformer
-from transform.views.image_filters import get_env, format_date
+import itertools
+import json
+import logging
+import os.path
+import shutil
+import subprocess
+import sys
+import zipfile
+
 from requests.packages.urllib3.exceptions import MaxRetryError
+
+try:
```

Comment:
Ah ok - is there any reason to be calling the module directly without the -m flag to notify that it is a script?